analysis_module/models.py

```diff
@@ -26,3 +35,59 @@ class FeaturesType(models.TextChoices):
 
     class Meta:
         db_table = "event_status_tracker"
+
+
+class FailedCallback(BaseModel):
+    class FailedStatus(models.IntegerChoices):
```

Comment:
Maybe use different name :D 
```suggestion
class CallbackTrack(BaseModel):
    class Status(models.IntegerChoices):
```

Reply:
FailedStatus -> Status sounds good and updated. The model name seems good to me. :D